src/js/components/Collapsible/__tests__/Collapsible-test.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import 'jest-styled-components';
+import { cleanup, render, fireEvent } from '@testing-library/react';
+
+import { Collapsible } from '..';
+import { Box } from '../../Box';
+import { Button } from '../../Button';
+import { Grommet } from '../../Grommet';
+import { Text } from '../../Text';
+
+describe('Collapsible', () => {
+  afterEach(cleanup);
+
+  test('open', () => {
+    const { container } = render(
+      <Grommet>
+        <Collapsible open>
+          <Box background="dark-1">Box Body</Box>
+        </Collapsible>
+      </Grommet>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('rerender-default', () => {
+    const useEffect = jest.fn();
+    let open = false;
+    const { container, rerender, getByText } = render(
+      <Grommet>
+        <Button
+          onClick={() => {
+            open = !open;
+            useEffect();
+          }}
+          label="Toggle"
+        />
+        <Collapsible id="hidden-test" open={open}>
```

Comment:
Is the `id` needed?

Reply:
No. Removed it in the most recent commit